pkg/migrations/migration_version.go

```diff
@@ -55,6 +61,29 @@ func Read(dbPath string) (*MigrationVersion, error) {
 	return version, nil
 }
 
+// ReadVersion - reads the version from the postgres database.
+func ReadVersion(pool *pgxpool.Pool) (*MigrationVersion, error) {
```

Comment:
I need to be able to read/write the version from migrator and from pkg/migrations.  It didn't make a lot of sense to keep the migration version in a file on central with the database deployed some places else entirely as that caused lots of complications.  So we are reading and writing to the version table which is what I believe its purpose to be.  So to do that I moved the postgres version store to pkg.  That may or may not have been the correct thing to do.  In doing so, I fail style checks here and in the store because of the import for resources.Version.  Do I need to move that to pkg as well?  Should I not move the store?  Thoughts on how best to proceed there?

Reply:
Yes, I got you. You can move the version store to pkg and leave a read-only wrapper in central with all the resource and access control. We can let migrator take care all maintainence of version.